tapscript/script.go

```diff
@@ -25,7 +25,7 @@ type FundingScriptTree struct {
 // by a 2-of-2 MuSig2 output.
 func NewFundingScriptTree() *FundingScriptTree {
```

Comment:
since this is no longer in `tapchannel` we could indicate in the func name that this is channel specific